RecoEgamma/ElectronIdentification/python/Identification/heepElectronID_HEEPV60_80XAOD_cff.py

```diff
@@ -0,0 +1,102 @@
+from PhysicsTools.SelectorUtils.centralIDRegistry import central_id_registry
+
+# Common functions and classes for ID definition are imported here:
+from RecoEgamma.ElectronIdentification.Identification.heepElectronID_tools import *
```

Comment:
It is recommended to replace `import *` with specific imports to avoid importing unneeded modules.